cetmix_tower_server/models/cx_tower_server.py

```
@@ -1550,70 +1553,61 @@ def _run_python_code(
+        """
+        Prepare SSH command(s) for execution on a remote server.
+
+        Behavior:
+        - Without sudo: returns the original command string,
+          optionally prefixed with `cd {path} && ` if a path is provided.
+        - With sudo:
+          * If no_split_for_sudo=True and '&&' in command_code:
+            do not split; prepend `sudo -S -p ''` once to the entire command.
+          * Otherwise, split command_code on '&&', strip line-continuation backslashes,
+            and for each segment: prepend `sudo -S -p ''`.
+            For passwordless sudo (mode 'n'), rejoin segments using '&&' into a single string.
```

Ruff (0.8.2)

Line too long (94 > 88)

(E501)